incubator/virtualcluster/pkg/syncer/resources/service/uws.go

```diff
@@ -54,6 +55,20 @@ func (c *controller) BackPopulate(key string) error {
 		return err
 	}
 
+	// Make sure the super cluster IP is added to the annotation so that it can be back populated to the tenant object
+	if pService.Spec.ClusterIP != "" && (pService.Annotations == nil || pService.Annotations[constants.LabelSuperClusterIP] != pService.Spec.ClusterIP) {
```

Comment:
just `pService.Annotations[constants.LabelSuperClusterIP] != pService.Spec.ClusterIP`

it won't panic even if pServer.Annotations is nil

Reply:
You are right. Done